core-concepts/src/components/Counter/Counter.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+import './Counter.css'
+
+class Counter extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            number: 0
+        }
+        this.decreaseNumber = this.decreaseNumber.bind(this)
+        this.increaseNumber = this.increaseNumber.bind(this)
```

Comment:
Just a small note, instead of binding you can use arrow functions.
In this case you don't have to bind the functions in the constructor.
Eg.:
    decreaseNumber = () => {
        this.setState({number: this.state.number - 1})
    }